### ISSUE TITLE
Peer-to-Peer code review

### ISSUE THREAD

**Issue by blessedjasonmwanza** [phelian23/awesome-books#5]
## Peer-to-Peer code review

Awesome work so far teams! 👍🏽 💯  It's amazing to see how well you collaborated to make this project come to life.
Here are a few components or suggestions that the review team proposes for you considerable implementation and improvements.

- The [readme file](https://github.com/phelian23/awesome-books/blob/e195ecb4829bd333e1f45bca1a22be41ef3f3c77/README.md?plain=1#L6-L10) does not have the screenshot of the project. To help users get that sneak peek of the project, kindly consider fixing this 🤓 .
- Furthermore, it seems like you currently do not have a live demo link for this project. You may consider deploying it using [GitHub pages](https://pages.github.com/), to help new project visitors have a live interaction with the project. 
- For improved form validation prior to adding the information to your Database, the team recommends that you add ```required``` attributes to your form [input fields](https://github.com/phelian23/awesome-books/blob/e195ecb4829bd333e1f45bca1a22be41ef3f3c77/index.html#L17-L18)
- For Data integrity purposes, the team recommends that you add a checker function that checks if the book ID ( currently generated using the [random function](https://github.com/phelian23/awesome-books/blob/e195ecb4829bd333e1f45bca1a22be41ef3f3c77/script.js#L73) ) already exists before adding it to the list to help avoid having duplicate IDs for different books.
- In a nutshell and as may be guided by the ```DRY``` principle, you may as well consider wrapping the [```display: flex```](https://github.com/phelian23/awesome-books/blob/e195ecb4829bd333e1f45bca1a22be41ef3f3c77/style.css#L19) into a class for improved code reusability.

> HappyCoding 👨🏽‍💻 👩🏽‍💻
